Update manual deployment doc (#1087)
btcpayserver/btcpayserver-doc: docs/Deployment/ManualDeployment.md

```bash
cd ~/NBXplorer
./run.sh --dbtrie
```

> Note that using the `--dbtrie` backend for NBXplorer, while being easier, this is deprecated.

Reviewer: ```suggestion
Note that using the `--dbtrie` backend for NBXplorer, while easier, is deprecated.
```

You should better use a postgresql backend as documented on [Extended Manual Deployment](./ManualDeploymentExtended.md).

> ### 7) Run BTCPayServer

Reviewer: ```suggestion
### 7) Run BTCPay Server
```

Now you can browse your server on port 8080.

> Note that by default, btcpayserver will use SQLite as backend, while being easier, this is deprecated.

Reviewer: ```suggestion
Note that by default, BTCPay Server will use SQLite as backend, while being easier, this is deprecated.
```
